app/src/androidTest/java/com/duckduckgo/rules/TurnOffAnimationRule.kt

```diff
@@ -42,4 +45,11 @@ class TurnOffAnimationRule : Instrumentation() {
     private fun putDeviceSetting(key: String, value: Int) {
         device.executeShellCommand("settings put global $key $value")
     }
+
+    override fun apply(
+        base: Statement?,
+        description: Description?
+    ): Statement {
+        TODO("Not yet implemented")
```

Comment:
To chyba jest niepotrzebne? 🤔

Reply:
To zostało dodane automatycznie przy implementacji funkcji "apply". Po usunięciu AS krzyczy, że brakuje returna.